sci-physics/thepeg/thepeg-2.3.0.ebuild

```
@@ -0,0 +1,114 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools elisp-common java-pkg-opt-2
+
+MY_P=ThePEG-${PV}
+
+DESCRIPTION="Toolkit for High Energy Physics Event Generation"
+HOMEPAGE="
+	https://home.thep.lu.se/ThePEG/
+	https://thepeg.hepforge.org/
+"
+
+TEST_URI="https://www.hepforge.org/archive/lhapdf/pdfsets/current"
+SRC_URI="https://www.hepforge.org/archive/thepeg/${MY_P}.tar.bz2
+	test? (
+		hepmc3? (
+			${TEST_URI}/cteq6ll.LHpdf
+			${TEST_URI}/cteq5l.LHgrid
+			${TEST_URI}/GRV98nlo.LHgrid
+			${TEST_URI}/MRST2001nlo.LHgrid )
+	)"
+
+LICENSE="GPL-2"
```

It's `GPL-3` now. Please see the [COPYING](https://phab.hepforge.org/source/thepeghg/browse/release-2-3/COPYING) and [README](https://phab.hepforge.org/source/thepeghg/browse/release-2-3/README).
```suggestion
LICENSE="GPL-3"
```

---

Previous versions 2.2.2 and 2.2.3 are also GPL-3 -> updated.